### PR TITLE
Exclude include 

### PR DESCRIPTION
**Pull request by DeliciousBounty**
This PR solved the following issue: #78  ( Users can have now the possibility to disable specific checks).
Based on a previous PR #114  but added the possibility to passive include, active include.
In addition I changed the profile intrusive to active and implemented it.

### CODE REVIEW

File: cherrybomb-engine/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cherrybomb-oas = "^0.1"
+#cherrybomb-oas = "^0.1"
+cherrybomb-oas={path = "../cherrybomb-oas"}
```
Comment (GuyL99) [inline cherrybomb-engine/Cargo.toml:17]: crates.io version needed

File: cherrybomb-engine/src/lib.rs [hunk 2]
```diff
@@ -9,17 +9,35 @@ use crate::scan::active::active_scanner;
...
+        }
+    }
+
+    if !config.active_exclude.is_empty() {
+        for active_check in config.active_exclude.iter() {
+            checks.remove(active_check);
+        }
+    }
+    checks.clone()
+}
```
Comment (GuyL99) [inline cherrybomb-engine/src/lib.rs:40]: checks.iter().filter_map(|check| if confuig).collect()

**Review by GuyL99 — CHANGES_REQUESTED** [review]

File: cherrybomb-engine/src/lib.rs (hunk 2 above)
Comment (GuyL99) [reply cherrybomb-engine/src/lib.rs:40]: put in config.rs in cherrybomb engine

**Review by GuyL99 — APPROVED** [review]
looks much better now

**Comment by jayvdb** [comment]
@DeliciousBounty , this branch now has conflicts

**Pull request merged**